canonical-web-and-design/jaas-dashboard
# Update dependencies

## Done

Upgraded the outdated dependencies.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Log in, via the link in the redux store.
- Everything should work as expected & CI should pass

## Details

There was some additional work required to upgrade `vanilla-framework` so a follow-up issue has been created #63 

Fixes #62
